pint/quantity.py

```diff
@@ -78,7 +78,7 @@ def wrapped(self, *args, **kwargs):
 
 
 @fix_str_conversions
-class _Quantity(PrettyIPython, SharedRegistryObject):
+class BaseQuantity(PrettyIPython, SharedRegistryObject):
```

Comment:
Why is this classe "public" (i.e. no underscore)? Is this the class users should check against when using `isinstance`?

Reply:
Hadn't thought of this when going through it! 

It should be used when the user does know the UnitRegistry. In the current version users should be checking with _Quantity (as done in pint-pandas) 
Using isinstance with a Quantity from one UnitRegistry and another UnitRegistry yields false:
'''
ureg=pint.UnitRegistry()

ureg2 = pint.UnitRegistry()

isinstance(my_array,ureg2.Quantity)
False
'''
Should this be public?